Tools/Scripts/webkitpy/w3c/test_downloader.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, repository_directory, host, options):
         self._test_suites = []
 
         self.repository_directory = repository_directory
+        self.upstream_commitID_head = None
```

Comment:
this is a weird mix of casing? `upstream_commit_id_head` would make more sense here, but still seems a bit wordy.

that said, I'd rather just go for something shorter like "upstream_revision"?